app/Auth0/Repositories/EloquentAuth0ClientRepository.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Auth0\Repositories;
+
+use App\Auth0\Auth0Client;
+use App\Auth0\Auth0ClientsForIntegration;
+use App\Auth0\Models\Auth0ClientModel;
+use Illuminate\Support\Facades\DB;
+use Ramsey\Uuid\UuidInterface;
+
+final class EloquentAuth0ClientRepository implements Auth0ClientRepository
+{
+    public function save(Auth0Client ...$auth0Clients): void
```

Comment:
Would it make sense to change the type here to `Auth0ClientsForIntegration`?

Reply:
I thought about it when refactoring it from saving one client to multiple. I decided to keep it generic, in the sense that the repository can save one or more `Auth0Client` regardless of how they were loaded/created. But I can change it if you prefer `Auth0ClientsForIntegration`. I'd rename the method then though

Reply:
Keep it, it's fine.